### PR TITLE
Port mapping was reversed for topo web server in docker-local

### PR DESCRIPTION
**Pull request by evanb**
# Pull Request Template

## Description

The port mapping for the web server in docker-local was reversed, so it was inaccessible from host.  Works now on http://localhost:3000/.

## Checklist:

- [y ] I have run the pre-commit checks before committing
- [y] I have performed a self-review of my own code
- [-] I have commented my code, particularly in hard-to-understand areas
- [-] I have made corresponding changes to the documentation
- [y] My changes generate no new warnings
- [-] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by joesustaric** [comment]
@evanb I killed the build and deploy steps. Well commented them out for now.

Build was creating the docker Images and pushing them to an aws ECR
Deploy was deploying it somewhere. (? Didn't look at all where it was going)

Thinking that the tests might be the most important thing at this stage.. We could re introduce the other steps at a later stage.. 

If you disagree and want to fix the rest.. Revert away!

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-    steps:
-      - checkout
-      - setup_remote_docker
-      - run:
-          name: Build
-          command: |
-            sudo pip install awscli
-            eval $(aws ecr get-login --no-include-email)
-            scripts/build
+  # build:
```
Comment (joesustaric) [inline .circleci/config.yml:3]: commented these out as we might want to fix these at a later stage.

File: .circleci/config.yml [hunk 2]
```diff
@@ -24,6 +24,10 @@ jobs:
             - node-dependency-cache-{{ checksum "yarn.lock" }}
             - node-dependency-cache-
       - setup_remote_docker
+      - run:
+          name: Install Dependencies
+          command: |
+            yarn install
```
Comment (joesustaric) [inline .circleci/config.yml:30]: This fixes the build when the cache disappeared.  We need to yarn install.. CircleCI seems to only keep caches around for 30days. 
This was probably removed in error.

**Comment by evanb** [comment]
This is weird - I didn't get notifications for the @mentions from github (or the comments on my PR).  Meh maybe I turned them off.

This looks good - I seem to recall @benkt removing some of the AWS infrastructure when we hibernated the project.  The tests running in the pipeline is all we need now.

**Pull request merged**